addons/payment_stripe/controllers/main.py

```diff
@@ -19,9 +18,46 @@
 
 class StripeController(http.Controller):
     _checkout_return_url = '/payment/stripe/checkout_return'
+    _onboarding_return_url = '/payment/stripe/onboarding/return/'
+    _onboarding_refresh_url = '/payment/stripe/onboarding/refresh/'
     _validation_return_url = '/payment/stripe/validation_return'
     WEBHOOK_AGE_TOLERANCE = 10*60  # seconds
 
+    @http.route(
+        _onboarding_return_url + '<int:acquirer_id>/<int:menu_id>', type='http', auth='public',
+        csrf=False
+    )
+    def stripe_onboarding_return(self, acquirer_id, menu_id):
+        """Stripe return URL
+
+            This route is used by Stripe to return after or during the onboarding.
+            Once called, the controller redirects to the payment acquirer.
+
+            :returns: Redirection to Stripe Acquirer Form
+        """
+        action = request.env.ref(
+            'payment_stripe.action_payment_acquirer_onboarding', raise_if_not_found=False
+        )
+        return request.redirect('/web?#action=%s&id=%s&menu_id=%s' % (
+            action and action.id or 'payment.action_payment_acquirer', acquirer_id, menu_id
+        ))
+
+    @http.route(
+        _onboarding_refresh_url + '<int:acquirer_id>/<account_id>/<int:menu_id>', type='http',
+        auth='public', csrf=False
```

Comment:
`csrf=False` is useful for post requests. Is that the case?

Reply:
No, it was a sign of cargo-cult computing, I just read HttpRequest.dispatch() right now and understand the meaning. Mea culpa and thanks for the useful info :)

Reply:
No, it was a sign of cargo-cult computing, I just read HttpRequest.dispatch() right now and understand the meaning. Mea culpa and thanks for the useful info :)